### DISCUSSION TITLE
Are there any EU dimmers left that come with Tasmota or that can be flashed to run it?

### DISCUSSION THREAD

**Discussion by markusdd** [arendst/Tasmota#22028] [Q&A]
Question is basically in the title.
A friend is searching for one and I tried to help him on that and I came back empty-handed.
Athom has discontinued theirs, and everythign in blakkadders repsoitory is either unavailbe or now incompatible due to other Wifi modules used by Tuya.

I would even go through the length of exchanging modules to ESP ones if needed, but I cannot even find a base model to do that with.

Any help?

**Comment by barbudor** [comment]
What about Shelly Dimmer 2 ?
But I recommend to use with neutral

**Comment by ferbulous** [comment]
For swapping ESP module, there's FS-05R

https://www.elektroda.com/rtvforum/topic4039890.html

You can use PWM dimmer for the buttons
